x-pack/legacy/plugins/apm/public/components/shared/KueryBar/index.tsx

```diff
@@ -120,15 +120,19 @@ export function KueryBar() {
     let didCancel = false;
 
     async function loadIndexPattern() {
-      setState({ ...state, isLoadingIndexPattern: true });
+      setState(value => ({ ...value, isLoadingIndexPattern: true }));
```

Comment:
Better 👍

Reply:
... again: did the linter catch this? If so, that's great!

Reply:
yep!